### ISSUE TITLE
All node debug breakpoints fire from app.js

### ISSUE THREAD

**Issue by Andrew-Max** [balderdashy/sails#3047]
Possibly related to https://github.com/balderdashy/sails/issues/2670 although I don't think so.

In a sense they are related though because if I could get those breakpoints working I wouldn't be pursuing the node debug route.

Basic reproduction steps: 

running

    $ node debug app.js

On a brand new sails app always breaks for me  at:
```
< Debugger listening on port 5858
debug> . ok
break in app.js:23
 21 // Ensure we're in the project directory, so relative paths work as expected
 22 // no matter where we actually lift from.
>23 process.chdir(__dirname);
 24 
 25 // Ensure a "sails" can be located:
```

Not sure if this is even a sails issue or possibly a node issue.

Just looking for some way to get break points working

**Comment by sailsbot** [comment]
Thanks for posting, @Andrew-Max.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
